ambari-logsearch/ambari-logsearch-web/src/app/components/main-container/main-container.component.less

```diff
@@ -17,5 +17,5 @@
  */
 
 :host {
-  overflow-x: hidden;
+  
```

Comment:
This file (and reference to it in the component decorator as well) isn't needed anymore